packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -222,8 +222,13 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
 
   @autobind
   private _onGotoToday() {
-    this._navigateDay(this.props.today!);
+    let { onSelectDate, today } = this.props;
+    this._navigateDay(today!);
     this._focusOnUpdate = true;
+
+    if (onSelectDate) {
```

Comment:
Also see my other comment on the conversation tab about the issue I found on the single month picker.